packages/components/psammead-grid/README.md

```diff
@@ -53,25 +76,72 @@ When we refer to `group3` in this component, we're referring to the breakpoint w
 
 #### <a name="simple-example">Simple example</a>
 
-Here is an example of a `<Grid>` that has 8 columns for `group4` (from 1008px to 1279px). It has two child `<Grid item>` elements, one which spans 6/8 columns and one which spans 2/8 columns within this breakpoint.
-
-In this example groups 1, 2, 3, & 5 are omitted from the `columns` prop - so at those breakpoints items fill the whole width of the page.
+Here is an example of a `<Grid>` that has 8 columns for `group4` (from 1008px to 1279px). It has four child `<Grid item>` elements, one which spans 6/8 columns and one which spans 2/8 columns within this breakpoint.
 
 ```jsx
 import Grid from '@bbc/psammead-grid';
 
 const MyComponent = () => (
-  <Grid columns={{ group4: 8 }}>
-    <Grid item columns={{ group4: 6 }}>
+  <Grid
+    columns={{
+      group0: 6,
```

Comment:
To keep this as a simple example, it'd be good to ensure it matches the description above - so has groups 0-5 as `8` on the outer grid and `6` / `2` for the relevant Grid items.